.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -14,9 +14,6 @@ Forum for questions: https://users.rust-lang.org/c/ide/14
 Before submitting, please make sure that you're not running into one of these known issues:
 
  1. on-the-fly diagnostics are mostly unimplemented (`cargo check` diagnostics will be shown when saving a file)
- 2. some platform-specific imports are not resolved: #6038
- 3. attribute proc macros are not supported: #6029
```

Comment:
attribute proc macros aren't enabled by default yet, so I'm not sure we should remove this already